src/composables/use-event-listener.js

```diff
@@ -9,7 +9,7 @@ import {
 /**
  * Use an event listener. Shamelessly stolen from https://logaretm.com/blog/my-favorite-5-vuejs-composables/#useeventlistener
  *
- * @param {import('@nuxtjs/composition-api').Ref<EventTarget | null> | EventTarget} target The target can be a reactive ref which adds flexibility
+ * @param {import('./types').MaybeRef<EventTarget | null> | EventTarget} target The target can be a reactive ref which adds flexibility
```

Comment:
I'm not sure where `MaybeRef` was introduced but I think the type for it is wrong. I'll open a bug ticket to fix that. In the mean time, just so that I understand the intention for it, is it meant to make it unnecessary to add `null` or `undefined` to the inner type?

Also before I open the issue, just to verify the type. Should it be this instead:

```js
/**
 * @template T
 * @typedef {import('@nuxtjs/composition-api').Ref<T | null | undefined>}
 */
```

Cause right now the type of it will result in a variable that is either a Ref<T> or T itself (which doesn't seem to match the name of `MaybeRef` and I can't figure out a use case for it).

Reply:
I copied this type from Vueuse: https://github.com/vueuse/vueuse/blob/8782e1761203cdc0bf41e95e38504fed98351c77/packages/shared/utils/types.ts#L15, 
And yes, it is a type that's either a ref or just a simple value.